### PR TITLE
Update: made changes highlighted in the Google doc

### PR DESCRIPTION
**Pull request by Goku-kun**
This PR fixes the following issues from the Google Doc review:

- Paragraph 3 talks about pulling up the Plotly Express documentation but no instructions in the specific sections to refer to specific methods when calling them.
- Break apart and [explain each method](https://docs.python.org/3/library/io.html#binary-i-o) in the `pd.read_csv(io.BytesIO(uploaded[file_name.csv]))` or simply declare another variable to store the output of `io.BytesIO(uploaded[file_name.csv])` and explain what is the content of the variable.

One more issue left from the Google Doc review:

- Link the colab docs about the `from google.colab import files` part. (No documentation about this part specficially)

But, I found the documentation about "how to upload/download files on Colab" and have linked it in.

### CODE REVIEW

File: projects/visualize-youtube-data-with-plotly/visualize-youtube-data-with-plotly.mdx [hunk 1]
```diff
@@ -123,6 +123,14 @@ df = pd.read_csv(io.BytesIO(uploaded["most_subscribed_youtube_channels.csv"]))
 display(df)
 ```
 
+Let’s break down and understand what the `df = pd.read_csv(io.BytesIO(uploaded["most_subscribed_youtube_channels.csv"]))` piece of code is doing.
+
+We know that `uploaded` is a dictionary with file names as keys and file data as values. So, `uploaded["most_subscribed_youtube_channels.csv"]` is simply getting the binary data inside the file named `most_subscribed_youtube_channels.csv` that we uploaded to Google Colab.
```
Comment (sonnynomnom) [inline projects/visualize-youtube-data-with-plotly/visualize-youtube-data-with-plotly.mdx:128]:
```suggestion
We know that `uploaded` is a dictionary with file names as keys and file data as values. So, `uploaded["most_subscribed_youtube_channels.csv"]` is simply getting the binary data inside the file named **most_subscribed_youtube_channels.csv** that we uploaded to Google Colab.
```

File: projects/visualize-youtube-data-with-plotly/visualize-youtube-data-with-plotly.mdx [hunk 2]
```diff
@@ -123,6 +123,14 @@ df = pd.read_csv(io.BytesIO(uploaded["most_subscribed_youtube_channels.csv"]))
...
+
+Python's `io` module manages file-related input and output operations. So, `io.BytesIO()` receives all the binary data and creates a stream of data. `pd.read_csv()` can't directly process binary data so we need to convert it into a stream before we can make a DataFrame from this data.
+
+This stream of data is finally read by `pd.read_csv()` method and turned into a DataFrame. We store this DataFrame in the `df` variable.
```
Comment (sonnynomnom) [inline projects/visualize-youtube-data-with-plotly/visualize-youtube-data-with-plotly.mdx:132]:
```suggestion
This stream of data is finally read by the `pd.read_csv()` method and turned into a DataFrame. We store this DataFrame in the `df` variable.
```

File: projects/visualize-youtube-data-with-plotly/visualize-youtube-data-with-plotly.mdx [hunk 3]
```diff
@@ -182,6 +190,8 @@ The first thing we want to do is pay attention to our axes. On the x-axis, we ha
 
 Also, a quick shoutout to the channel all the way at the end that made it to 200 million ([T-Series](https://www.youtube.com/aashiqui2)). We already know their fan base is grinding.
 
+Plotly Express allows us to create much more customizable histograms by inputting a variety of arguments. Here's a link to [histogram documentation](https://plotly.com/python/histograms/) for further reference. In this section, we only used the base version with numerical data. But other histograms can also be created based on other columns in the YouTube data.
```
Comment (sonnynomnom) [inline projects/visualize-youtube-data-with-plotly/visualize-youtube-data-with-plotly.mdx:193]:
```suggestion
Plotly Express allows us to create much more customizable histograms by inputting a variety of arguments. Here's a link to [histogram documentation](https://plotly.com/python/histograms/) for further reference. In this section, we only used the base version with numerical data. But other histograms can also be created using other columns in the YouTube data.
```

**Review by sonnynomnom — COMMENTED** [review]
Left some suggestions @Goku-kun! Take a look and then merge it!

**Review by Goku-kun — COMMENTED** [review]
Okay. I've committed the suggestions and I'm merging this PR now.

**Pull request merged**